### PR TITLE
[FIX] Broken link in readme and license fix (& update)

### PR DESCRIPTION
**Pull request by Juknum**
### Changements 
- J'ai mis à jours la license présente dans chaque fichier en la réduisant au mieux et en redirigeant ce qui était superflu sur le fichier de license
- J'ai fix le tag du Discord dans le README

### CODE REVIEW

**Review by TheoDurr — APPROVED** [review]

File: LICENSE.old [hunk 1]
```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```
Comment (Hyask) [inline LICENSE.old:1]: This file can't be removed without explicit authorization of the original author.

File: accounting/__init__.py [hunk 2]
```diff
@@ -1,23 +1,15 @@
 # -*- coding:utf-8 -*
 #
-# Copyright 2016,2017
-# - Skia <skia@libskia.so>
```
Comment (Hyask) [inline accounting/__init__.py:4]: I'm not sure you're allowed to delete the original author of this file either. Adding the subsequent authors under the `AE UTBM` name should be OK, but you just can't erase the past like this.

**Review by Hyask — COMMENTED** [review]
This change is a shame. The very least you could have done is ask the concerned original authors about your wish to make the change, but do that silently while you still have contact with them shouldn't even have crossed your mind.

**Comment by Hyask** [comment]
@Juknum @TheoDurr there has been much activity on the repo since my messages, but no answer on this PR, so I'll highlight you, and ask you to please respond to this message.

What you just did with this change is basically illegal. You are still in your student years, so this can be forgivable because you have the benefit of the doubt: you probably didn't know how to handle this, and made the change without thinking evil. Mistakes are human, but humans also shall correct their mistakes. To repair the legal damage you caused, you can:
- restore the `LICENSE.old` file containing the original MIT license. This file just cannot be removed from the sources. I apologize for changing the license at the start of the project back in the time, but once an MIT license is applied to a source code, the file must remain in the sources even when a re-licensing occurs.
- restore the license owner at the top the files you changed.  
  If your concern is about a growing list of names in some files, a possibility for you is to just add the `AE UTBM` as copyright owner (basically as you did) for present and future years, to stop the list from growing, and make a contributor agreement stating that people contributing to the code leave their copyright to `AE UTBM`. A nice thing to do in that case is add a `CONTRIBUTORS` file at the root of the repo to list the contact of people that contributed.

GPL is a very widespread license that is used for thousands of open-source software projects. Learning how it works, what you can or cannot do with it is something important for your future work in the field of software development. Violating well-known licenses publicly before even the start of your career isn't something you should take lightly, as this could damage your reputation regarding future employers.

I really advise you to answer this message before doing future work on this project, as this would prove that you really don't care about the problem. A kind solution found quickly will always be preferable to hundreds of messages exchanged on public platforms to argue about an issue that is obviously on your side.

I hope you understand and will try to cooperate :-)

**Pull request merged**